konfera/tests/test_models.py

```diff
@@ -178,6 +178,27 @@ def test_string_representation(self):
 
 
 class TicketTest(TestCase):
+    def setUp(self):
+        time = timezone.now()
+        location = models.Location(title='test_title', street='test_street', city='test_city', postcode=000000,
```

Comment:
I know this *bug* was here before but `postcode` is a `CharField` so `'00000'` would be better.